docs/docs/platforms/openshift/OPENSHIFT.md

```diff
@@ -0,0 +1,100 @@
+
+However, the default OpenShift **restricted** Security Context Constrain ([SCC](https://docs.openshift.com/container-platform/4.14/authentication/managing-security-context-constraints.html){.external}) requires pods to be run with a UID, and SELinux context that are allocated to the namespace. It means it is not possible to deploy DC Helm charts with the default settings (with pre-defined `securityContext` and containers that run as root). See [FAQs](../OPENSHIFT_FAQ) for typical errors.
+
+!!! warning "ATL_TOMCAT_* Environment Variables"
+    When running as restricted or nonroot SCC, some of `ATL_TOMCAT_*` environment variables passed to the Helm chart in `additionalEnvironmentVariables` will be ignored, because `server.xml` and  `seraph-config.xml` are mounted as a ConfigMaps rather than generated in the container entrypoint from Jinja templates. Both `<product>.tomcatConfig` and `<product>.seraphConfig` have a number of properties which you can override if necessary. Look at `tomcatConfig` stanza in the chart's values.yaml for more details.
+
+=== "Restricted SCC (:octicons-thumbsup-16: Recommended)"
+    !!! info "Restricted SCC"        
+        Restricted SCC denies access to all host features and requires
+        pods to be run with a UID, and SELinux context that are allocated to the namespace.
+    
+    Running containers in OpenShift with a restricted SCC is the most secure approach. To be able to successfully deploy Atlassian DC Helm charts with a restricted SCC, set `openshift.runWithRestrictedSCC` to true in Helm values file:
+
+    ```yaml
+    openshift:
+      runWithRestrictedSCC: true
+    ```
+
+    This property will:
+    
+    * unset `securityContext` on the pod and `initContainers` level, letting OpenShift set it
+    * trigger creation of a ConfigMap with `server.xml` and `seraph-config.xml` that are mounted as read-only files into containers (rather than generated in the container entrypoint). Both `server.xml` and `seraph-config.xml` can be configured in `<product>.tomcatConfig` Helm stanza (except for Bitbucket which does not have these configuration files)
+    * disable nfs-permission fixer init container that is run as root by default. If you need to change permissions in an existing shared-home volume, do it when provisioning/migrating the volume. See [FAQs](../OPENSHIFT_FAQ/#nfs-permission-fixer-is-disabled-how-can-i-make-shared-home-writable) for more details.
+
+=== "Nonroot SCC"
+    !!! info "Nonroot SCC"
+          Nonroot SCC provides all features of the restricted SCC but allows users to run with **any** non-root UID.
+
+    !!! info "Supported Versions"
+        OpenShift nonroot friendly values are available in versions `1.14.0+` for Jira and Confluence, and `1.18+` for Crowd and Bamboo. If you can't upgrade, consider using 'additionalFiles' to mount `server.xml` and `seraph-config.xml` as ConfigMaps which should be created outside the Helm chart.
```

Comment:
Does "nonroot friendly values" mean values that are friendly to the non-root security context? Or are they "friendly values"? I'm not familiar with this term, but if I was right first time, this should be written as "OpenShift non-root-friendly values"